### PR TITLE
Fix python build

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

**Review by atobiszei — APPROVED** [review]

File: .bazelrc [hunk 1]
```diff
@@ -109,3 +109,11 @@ test --define=MEDIAPIPE_DISABLE=1
 build --cxxopt=-DMEDIAPIPE_DISABLE=1
 coverage --cxxopt=-DMEDIAPIPE_DISABLE=1
 test --cxxopt=-DMEDIAPIPE_DISABLE=1
+
+build --define=PYTHON_DISABLE=1
+coverage --define=PYTHON_DISABLE=1
+test --define=PYTHON_DISABLE=1
+
```
Comment (atobiszei) [inline .bazelrc:116]: We could add flag MEDIAPIPE_DISABLE here as well?

Eventually we should switch this flag name to OVMS_MEDIAPIPE_DISABLE to be able to upstream at some point.

Comment (atobiszei) [reply .bazelrc:116]: Beside that LGTM

Comment (rasapala) [reply .bazelrc:116]: flag is added before this line.

**Review by dtrawins — APPROVED** [review]

**Comment by rasapala** [comment]
Already commited.

**Pull request closed without merging**